start/node/src/main/java/org/ethereum/beacon/node/NodeCommandLauncher.java

```diff
@@ -189,12 +183,15 @@ protected ThreadFactory createThreadFactory(String namePattern) {
         config.getConfig().getValidator().getSigner(),
         config.getChainSpec().getSpecHelpersOptions().isBlsSign());
 
-    ConnectionManager<?> connectionManager;
     if (config.getConfig().getNetworks().size() != 1) {
       throw new IllegalArgumentException("1 network should be specified in config");
     }
     Network networkCfg = config.getConfig().getNetworks().get(0);
+    Libp2pLauncher libp2pLauncher = new Libp2pLauncher();
+
     if (networkCfg instanceof NettyNetwork) {
+      throw new UnsupportedOperationException("Netty network is not supported anymore");
```

Comment:
I will sort it out a bit later. May be it makes sense to get rid of NettyNetwork stuff completely, since old Netty and new Libp2p are pretty different in terms of interfaces